_test/helper.test.php

```diff
@@ -0,0 +1,83 @@
+<?php
+
+class helper_plugin_captcha_public extends helper_plugin_captcha {
+
+    public function get_field_in() {
+        return $this->field_in;
+    }
+    public function get_field_sec() {
+        return $this->field_sec;
+    }
+    public function get_field_hp() {
+        return $this->field_hp;
+    }
+}
+
+/**
+ * @group plugin_captcha
+ * @group plugins
+ */
+class helper_plugin_captcha_test extends DokuWikiTest {
+
+    protected $pluginsEnabled = array('captcha');
+
+    public function testConfig() {
+        global $conf;
+        $conf['plugin']['captcha']['lettercount'] = 20;
+
+        $helper = new helper_plugin_captcha_public();
+
+        $code = $helper->_generateCAPTCHA("fixed", 0);
+        $this->assertEquals(16, strlen($code));
+    }
+
+    public function testDecrypt() {
+
+        $helper = new helper_plugin_captcha_public();
+
+        $rand = "12345";
+        $secret = $helper->encrypt($rand);
+        $this->assertNotSame(false, $secret);
+        $this->assertSame($rand, $helper->decrypt($secret));
+
+        $this->assertSame(false, $helper->decrypt(''));
+        $this->assertSame(false, $helper->decrypt('X'));
```

Comment:
No particular reason. Too much copy-and-paste and I didn't think of making the test files look pretty.